src/libultra/gu/sins.c

```diff
@@ -2,6 +2,10 @@
 
 #include "sintable.inc.c"
 
+/**
+ * @param x binang
+ * @return sin(x)*0x7FFF
```

Comment:
I'll admit, I get where the `*0x7FFF` is coming from, but it seems kind of strange and did take me a little bit to think it through. I like what you did in `Math_GetAtan2Tbl`, so perhaps using `@return sin(x) as binang` instead?

Reply:
sin() doesn't return a binang :sunglasses: 
this is just a sin() value but since it's an integer value, it's scaled to go from `-short_max` to `short_max`

Reply:
Oh right, `sin` doesn't return an angle 🤦. I'm fine with this than.